plugins/PrivacyManager/Controller.php

```diff
@@ -329,6 +336,7 @@ private function getAnonymizeIPInfo()
         $privacyConfig = new Config();
         $anonymizeIP["enabled"] = IPAnonymizer::isActive();
         $anonymizeIP["maskLength"] = $privacyConfig->ipAddressMaskLength;
+        $anonymizeIP["forceCookielessTracking"] = $privacyConfig->forceCookielessTracking;
```

Comment:
when saving the setting, could we directly update the tracker? I think it didn't happen when I tested it and had to manually trigger the update.

Reply:
or maybe I didn't look correctly

Reply:
Actually that should already be done in the API method that saves the settings...